starter-code/contact-app/src/styles.css

```diff
@@ -1 +1,9 @@
 /* You can add global styles to this file, and also import other style files */
+body{
+    background: lightblue url("https://media.giphy.com/media/A5pcWMMIEO95S/giphy.gif") no-repeat fixed center;
+    outline :white solid 20px; 
+}
+img{
```

Comment:
Maybe you prefer to put this in 'contact-list.component.css', then the rules are applied only to the images in that component.